images/toolbox/toolbox.sh

```diff
@@ -28,7 +28,7 @@ KEYRING_FILE="/etc/ceph/keyring"
 # without specifying any arguments
 write_endpoints() {  
     endpoints=$(cat ${MON_CONFIG})
-    mon_endpoints=$(echo ${endpoints} | sed 's/rook-ceph-mon[0-9]=//g')
+    mon_endpoints=$(echo ${endpoints} | sed 's/rook-ceph-mon[0-9]*=//g')
```

Comment:
I think a `+` would be better because the endpoints should always contain a number in the name.

Reply:
agreed. i actually tried that and it didn't work. looks like i was missing an escape: `\+` so it's working now